helm_deploy/apply-for-legal-aid/templates/dashboard_total_submitted_applications_cron.yml

```diff
@@ -0,0 +1,36 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}-dashboard-total-submitted-applications
+  labels:
+    app: {{ template "apply-for-legal-aid.name" . }}
+    chart: {{ template "apply-for-legal-aid.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  schedule: '*/11 * * * *'
```

Comment:
Just checking that this cron schedule is what is intended - it will run every 11 minutes, which seems an odd period?

In general, all the jobs being created have different schedules:  every two minutes, every five minutes, every 10 minutes, every 11 minutes and every hour. Is there any benefit to doing it this way? Would it be clearer/easier to maintain if they all ran on the same schedule?

Reply:
Thanks, it's been updated to every 10 minutes to match the submitted application cron job which is required to be every 10 minutes. The cron jobs are not all run at the same time for best practice.